worldedit-bukkit/adapters/adapter-1.17.1/build.gradle.kts

```diff
@@ -1,3 +1,17 @@
-applyPaperweightAdapterConfiguration(
```

Comment:
I would just drop this folder entirely, we're removing all legacy adapters with 7.3.0, and that means 1.17 too.